<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_user</class>
 <widget class="QDialog" name="set_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cbx_icon">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>61</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lb_picture">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>61</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_signature">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>139</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>签名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_signature">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>139</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>217</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>217</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clear">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_emit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
